src/python/turicreate/test/_test_api_visibility.py

```diff
@@ -8,6 +8,7 @@
 from __future__ import absolute_import as _
 import unittest
 import turicreate
```

Comment:
This is an _ test. Does it actually run?

Reply:
Not by default. It does when we run:

```bash
 pytest _test_api_visibility.py 
```

Reply:
Most of the tests in here fail though.

Reply:
Can we make it run by default. Maybe a good time to actually consider doing it.

Reply:
I can update that file if we want those tests to pass. We just haven't been updating it every time we add new api.

Reply:
Done